README.md

```diff
@@ -1358,6 +1358,9 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [CodeMyUI](https://codemyui.com) - Handpicked collection of Web Design & UI Inspiration with Code Snippets.
   * [ColorKit](https://colorkit.co/) - Create color palettes online or get inspiration from top palettes.
   * [coolors](https://coolors.co/) - Color palette generator. Free.
+  * [Colors & Fonts](https://colorsandfonts.com/) -- Free. Color and typography resources and inspiration ready to copy paste
+  * [SVG Gradients](https://svggradients.com/) -- Free. A collection of 30 SVG Gradients ready to copy paste
+  * [SVG Dooldes](https://svgdoodles.com/) -- Free. A collection of 208 SVG doodles ready to copy paste
```

Comment:
same, this is just a way to drive traffic to ads sorry

Reply:
> same, this is just a way to drive traffic to ads sorry

all those projects are free.

this is not free by any means:
This link is paid, * [coolors](https://coolors.co/) - Color palette generator. Free.